### PR TITLE
MAINT: add pep8speaks config file

### PR DESCRIPTION
**Pull request by kohr-h**
Let's see what happens :-)

### CODE REVIEW

**Comment by kohr-h** [comment]
Hm, nothing.

File: .pep8speaks.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# File : .pep8speaks.yml
+
+message:  # Customize the comment made by the bot
+    opened:  # Messages when a new PR is submitted
```
Comment (adler-j) [inline .pep8speaks.yml]: Strongly against spamming all PRs like this. Better just leave a comment when something goes wrong.

File: .pep8speaks.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        header: "Hi @{name}, thanks for updating the PR!"
+        footer: ""  # Why to comment the link to the style guide everytime? :)
+    no_errors: "There are no PEP8 issues."
+
+scanner:
+    diff_only: False  # If True, errors caused by only the patch are shown
+
+pycodestyle:
+    max-line-length: 79  # Default is 79 in PEP8
+    ignore:  # Errors and warnings to ignore
```
Comment (adler-j) [inline .pep8speaks.yml]: I guess there are some in our settings?

Comment (kohr-h) [reply .pep8speaks.yml]: We have one in our settings, but it doesn't seem to apply anymore. It's E402:
```
relative-beyond-top-level (E0402): *Attempted relative import beyond top-level package*
```
It's not triggered anymore as it seems, so I'd just leave this section empty for now.

File: .pep8speaks.yml (hunk 1 above)
Comment (kohr-h) [reply .pep8speaks.yml]: Yeah, this is sort of the default, I'll replace it with an empty string.

**Comment by kohr-h** [comment]
Okay now it seems to work. The used config seems to be the last before pushing. I'll test again with the updated config.

**Comment by kohr-h** [comment]
Travis is also happy, no longer reporting PEP8 violations.
I'll check what happens if I update with a new PEP8 violation. The doc of pep8speaks says that there should be only one comment at a time, not a new one every time.

**Comment by kohr-h** [comment]
Testing now what happens when issues are fixed.

**Comment by kohr-h** [comment]
Ah, E402 is triggered in `__init__.py` files. I'll add it again.

**Comment by adler-j** [comment]
Looking good now I'd say!

**Comment by kohr-h** [comment]
This last push updated the comment above. So perhaps it's a good idea anyway to write *something* regardless of issues or not to make sure that the bot message is always close to the top.

I'll try pushing an update that fixes pep8, and see what happens to the comment.
The I'm going to rebase after that and see if it also works.

**Comment by kohr-h** [comment]
Okay, now it's updated obviously it uses the default config.

**Comment by kohr-h** [comment]
Okay, also the rebased push triggered an update. I think we can merge this now, perhaps fix small annoyances later.

**Comment by kohr-h** [comment]
Approve?

**Pull request merged**